docs/conf.py

```diff
@@ -88,7 +88,7 @@
     var home = "clouddocs.f5.com";
     var rtd = "f5-openstack-lbaasv2-driver.readthedocs.io";
 
-    if (window.location.hostname === rtd) {window.location.assign("http://" + home + "/products/openstack/lbaasv2-driver/liberty");}
+    if (window.location.hostname === rtd) {window.location.assign("http://" + home + "/products/openstack/lbaasv2-driver/stable/newton");}
```

Comment:
This is not correct. We are not using the 'stable' reference in the docs site paths.